finix/model/finix_list.py

```diff
@@ -0,0 +1,171 @@
+from typing import List
+
+
+# finix list implementation
+class FinixList:
+    def __init__(self, some_list, method, **kwargs):
+        self._method = method
+        self._kwargs = kwargs
+
+        if hasattr(some_list, 'embedded'):
+            tmp = some_list.embedded
+            if isinstance(tmp, dict):
```

Comment:
`if isinstance(some_list.embedded)` would work too, not strictly necessary to give some_list.embedded a new name.